components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -84,6 +84,13 @@ location.href = 'list-service-provider.jsp';
     
     if (samlIssuerName != null && "delete".equals(action)){
     	appBean.deleteSAMLIssuer();
+        //After deleting the SAMLIssuer from the ApplicationBean, need to remove the issuer data from the database.
```

Comment:
This part of the code is a boiler plate code to call axis2 services, which should not appear in multiple JSP pages. Need to use the same code in update page.